docs/docs/developers/tutorials/codealong/js_tutorials/aztecjs-getting-started.md

```diff
@@ -112,7 +112,7 @@ A successful run should show something like this:
   enr: undefined,
   nodeVersion: '0.82.0',
   l1ChainId: 31337,
-  protocolVersion: 1,
+  rollupVersion: 1,
```

Comment:
any thing needed in the migration notes?

Also I am adding this PR as a gating requirement for testnet - lmk if not

Reply:
done in https://github.com/AztecProtocol/aztec-packages/pull/13179